src/Model/Comment.php

```diff
@@ -104,6 +104,7 @@ class Comment extends DataObject
         'AuthorName' => 'Varchar',
         'RSSName' => 'Varchar',
         'DeleteLink' => 'Varchar',
+        'Date' => 'Datetime',
```

Comment:
What is the intended difference here between this native data field, and the new getter on line 872 that returns the `Created` data field?

Reply:
There isn't a native data field, this PR adds a getter that will handle calls to one though (such as in RSSFeed)

Reply:
oh whoops, this is a casting field isn't it :/ haha
...
yep 😂